pkg/webui/console/views/overview/index.js

```diff
@@ -247,6 +248,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={m.needHelp}
+                values={{
+                  documentationLink: (
+                    <Link.Anchor secondary href={documentationBaseUrl}>
+                      <Message content={sharedMessages.documentation} />
+                    </Link.Anchor>
+                  ),
+                  supportLink: (
+                    <Link.Anchor secondary href={supportLink}>
```

Comment:
We have an `external` prop for that:
```suggestion
                    <Link.Anchor secondary href={supportLink} external>
```